src/js/ia/Helpers.js

```diff
@@ -30,7 +30,7 @@
         var timestring = full? " days ago" : "d";
         if (date) {
             // expected date format: YYYY-MM-DDTHH:mm:ssZ e.g. 2011-04-22T13:33:48Z
-            date = date.replace("/T.*Z/", "");
+            date = date.replace(/T.*Z/, "");
```

Comment:
@jagtalon @jdorweiler The reason why we're stripping the hours, minutes and seconds is that this way it's easier to get an int number (just days) as difference, when calculating the elapsed time.
There's no reason to change this part.